### PR TITLE
FIX Remove legacy sprintf style translations in favour of placeholders

### PR DESCRIPTION
**Pull request by robbieaverill**
Issue: https://github.com/silverstripe/silverstripe-framework/issues/6582

### CODE REVIEW

**Comment by tractorcow** [comment]
@robbieaverill can you please rebase.

**Comment by robbieaverill** [comment]
@tractorcow rebased, thanks

File: src/VersionedGridFieldItemRequest.php [hunk 1]
```diff
@@ -172,7 +172,7 @@ public function doUnpublish($data, $form)
             $record->doUnpublish();
 
         $message = sprintf(
-            _t('SilverStripe\\Versioned\\VersionedGridFieldItemRequest.Unpublished', 'Unpublished %s %s'),
```
Comment (tractorcow) [inline src/VersionedGridFieldItemRequest.php]: Need to replace sprintf() with the parameterised form of `_t()` call.

Comment (robbieaverill) [inline src/VersionedGridFieldItemRequest.php]: D'oh. 2 secs

**Comment by flamerohr** [comment]
@tractorcow this was merged into `master` instead of the preferred `1` branch

**Comment by robbieaverill** [comment]
@flamerohr the other PRs are all in master too

**Comment by tractorcow** [comment]
I'll merge them back before tag.

**Pull request merged**